ios/Encryption/EncryptionUtil.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import CommonCrypto
+
+class EncryptionUtil {
+    enum Error: Swift.Error {
+      case generateRandomFailed
+      case getKeyFailed
+      case decryptionFailed
+    }
+
+    static let KEY_SIZE = kCCKeySizeAES256
+    static let HMAC_SIZE = Int(CC_SHA256_DIGEST_LENGTH)
+    
+    static let AES_KEY_TAG = "covidsafe.keys.aes_key"
+    static let HMAC_KEY_TAG = "covidsafe.keys.hmac_key"
+
+    static func encrypt(plainText: String) -> String {
+        do {
+            let aesKey = try KeyChain.get(tag: AES_KEY_TAG) ?? createKey(tag: AES_KEY_TAG)
+            print("AES key: \(aesKey.base64EncodedString())")
+      
+            let aes = try AES(key: aesKey)
+            var encryptedData: Data = try aes.encrypt(plainText)
+            print("String encrypted (base64): \(encryptedData.base64EncodedString())")
+
+            let hmacKey = try KeyChain.get(tag: HMAC_KEY_TAG) ?? createKey(tag: HMAC_KEY_TAG)
+            print("HMAC key: \(hmacKey.base64EncodedString())")
+            let hmac = try HMAC(key: hmacKey)
+            let hmacVal: Data = try hmac.getHMAC(encryptedData)
+            encryptedData.append(hmacVal)
+            print(encryptedData)
+            return encryptedData.base64EncodedString()
+        } catch {
+          print("Error: \(error)")
```

Comment:
The project is configured to use 2 spaces for indentation. Could you please reformat the files?